cmake/ConfigureCompiler.cmake

```diff
@@ -184,6 +193,11 @@ else()
       -Wno-unused-value
       -Wno-tautological-pointer-compare
       -Wno-format)
+    if (USE_CCACHE)
+      add_compile_options(
+        -Wno-register
+        -Wno-error=unused-command-line-argument)
```

Comment:
Why are these only enabled when USE_CCACHE is set?  I wouldn't expect using ccache to change what warnings occur.

Reply:
These warnings only occur when using ccache/distcc.
[https://stackoverflow.com/questions/47094843/clang-warning-argument-unused-during-compilation-stdlib-libc](url)